### PR TITLE
Adding start.cooking

### PR DESCRIPTION
**Pull request by 0x7183**
PR to add start.cooking

### CODE REVIEW

File: cms/apps/applications.json [hunk 1]
```diff
@@ -367,6 +367,18 @@
...
+    {
+      "title": "Start Cooking",
+      "subtitle": "Minting your Memecoin on Osmosis with just 2 clicks. All it takes is 1 OSMO.",
+      "external_URL": "https://www.start.cooking/",
+      "thumbnail_image_URL": "https://raw.githubusercontent.com/osmosis-labs/fe-content/main/cms/apps/images/start_cooking_thumbnail.png",
+      "hero_image_URL": "https://raw.githubusercontent.com/osmosis-labs/fe-content/main/cms/apps/images/start_cooking_hero.png",
+      "twitter_URL": "https://twitter.com/startcookingapp",
+      "medium_URL": "https://medium.com/@start.cooking157",
+      "github_URL": "https://github.com/BIG-Labs/cooking-contracts",
+      "featured": true
```
Comment (JeremyParish69) [inline cms/apps/applications.json:380]: Can we start with it not featured just yet?

Comment (0x7183) [reply cms/apps/applications.json:380]: > Can we start with it not featured just yet?

Hey, sorry for it, I missed the fact that featured was for internal use. It's ok for me

File: cms/apps/applications.json [hunk 2]
```diff
@@ -367,6 +367,18 @@
       "medium_URL": "https://medium.com/@whitewhaleterra",
       "github_URL": "https://github.com/White-Whale-Defi-Platform",
       "featured": false
+    },
+    {
+      "title": "Start Cooking",
+      "subtitle": "Minting your Memecoin on Osmosis with just 2 clicks. All it takes is 1 OSMO.",
+      "external_URL": "https://www.start.cooking/",
```
Comment (JeremyParish69) [inline cms/apps/applications.json:374]: Oh, this domain isn't registered? my DNS couldn't find it

File: cms/apps/applications.json [hunk 3]
```diff
@@ -367,6 +367,18 @@
...
+    },
+    {
+      "title": "Start Cooking",
+      "subtitle": "Minting your Memecoin on Osmosis with just 2 clicks. All it takes is 1 OSMO.",
+      "external_URL": "https://start.cooking/",
+      "thumbnail_image_URL": "https://raw.githubusercontent.com/osmosis-labs/fe-content/main/cms/apps/images/start_cooking_thumbnail.png",
+      "hero_image_URL": "https://raw.githubusercontent.com/osmosis-labs/fe-content/main/cms/apps/images/start_cooking_hero.png",
+      "twitter_URL": "https://twitter.com/startcookingapp",
+      "medium_URL": "https://medium.com/@start.cooking157",
+      "github_URL": "https://github.com/BIG-Labs/cooking-contracts",
```
Comment (JeremyParish69) [inline cms/apps/applications.json:379]: finding that the github link results in 404. Maybe it's private?

**Pull request merged**